specifyweb/frontend/js_src/lib/components/QueryBuilder/Export.tsx

```diff
@@ -100,9 +100,14 @@ export function QueryExportButtons({
     return resultToExport.join('\n');
   }
 
+  const containsResults =
+    (results.current?.length ?? 0) > 0 &&
```

Comment:
hmm, this line seems redundant with the next one
(.every() returns false for empty arrays)

Reply:
The type of results is `React.MutableRefObject<RA<QueryResultRow | undefined> | undefined>`, thus the line `results.current?.every((row) => row !== undefined)` handles the case when `results.current` is an array and contains elements, but may have `undefined` elements.

Reply:
ups, I was wrong about `(.every() returns false for empty arrays)` 🤦

.every() returns TRUE for empty arrays

it's `.some()` that returns false for empty

---

thus, in this case, could you use `results.current?.some(row => row !== undefined)` instead?

Reply:
Fixed in https://github.com/specify/specify7/pull/3590/commits/0a88e94fb07f3c92adf821397c97dca57f14d34e